### PR TITLE
Add Ethereum

### PR DESCRIPTION
**Pull request by himanshu1603**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

***********EDITING AN EXISTING TOPIC OR COLLECTION************

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this topic should be curated:

************CURATING A NEW COLLECTION************

- [ ] I've formatted my changes as a new folder directory, named for the collection as it appears in the URL on GitHub (e.g. `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this collection should be curated:

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/ethereum/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+logo: ethereum.png
+released: July 30, 2015
+short_description: Ethereum is a distributed public blockchain network.
+topic: ethereum
+related: blockchain, cryptocurrency
+url: https://www.ethereum.org/
+github_url: https://github.com/ethereum
+wikipedia_url: https://en.wikipedia.org/wiki/Ethereum
+---
+Ethereum is a decentralized platform that runs contracts based applications without any possibility of downtime, censorship, fraud or third-party interference. Ethereum blockchain focuses on running the programming code of any decentralized application.
```
Comment (MikeMcQuaid) [inline topics/ethereum/index.md]: - `contracts based` -> `contracts-based` or `contract-based`
- is it `running the programming code`, `writing the code` or `maintaining the code`?

Comment (himanshu1603) [reply topics/ethereum/index.md]: Is it wrong to use "running the programming code"?

Comment (MikeMcQuaid) [reply topics/ethereum/index.md]: @himanshu1603 `programming code` feels a bit weird/redundant to me; I think most people in the context of GitHub will assume `code` == `programming code`. What do you think?

Comment (himanshu1603) [reply topics/ethereum/index.md]: what about 'running the programs'?

Comment (MikeMcQuaid) [reply topics/ethereum/index.md]: @himanshu1603 In that case it'd just be `Ethereum blockchain focuses on running any decentralized applications.` Either that or my suggestion which was `Ethereum blockchain focuses on running the code of any decentralized application.`

File: topics/ethereum/index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+topic: ethereum
+related: blockchain, cryptocurrency
+url: https://www.ethereum.org/
+github_url: https://github.com/ethereum
+wikipedia_url: https://en.wikipedia.org/wiki/Ethereum
+---
+Ethereum is a decentralized platform that runs contract-based applications without any possibility of downtime, censorship, fraud or third-party interference. Ethereum blockchain focuses on running the code of any decentralized application.
+
+
+
```
Comment (MikeMcQuaid) [inline topics/ethereum/index.md]: Sorry, last thing: call you kill the extra whitespace? Then we're good to go!

**Comment by MikeMcQuaid** [comment]
Thanks again @himanshu1603!

**Comment by himanshu1603** [comment]
Thanks!

**Pull request merged**